src/_guides/language/analysis-options.md

```diff
@@ -293,12 +293,11 @@ linter:
     await_only_futures: true
 ```
 
-{{site.alert.note}}
-  **If you disable any rules, then use key-value syntax to enable rules.**
-  For example, the preceding example can’t use `- await_only_futures`;
-  instead, it must use `await_only_futures: true`.
-  The reason it can’t use the dash shortcut (`-`) to enable `await_only_futures`
-  is that YAML doesn't support mixing list and key-value syntaxes.
+{{site.alert.important}}
+  **Don't mix list and key-value syntaxes for the rules in the same options file.**
+  YAML doesn't support mixing both syntaxes in the same collection.
```

Comment:
Thoughts on this phrasing?

Reply:
Hi. Thanks for working on this issue. As for feedback ...

The inlined heading isn't accurate. Working from what you propose, how about something like this:

> **YAML doesn't support mixing both list and key-value syntax in the same rules entry.** 
> However, your are free to use any syntax for `rules` appearing in an included options file.

@kwalrath WDYT? Also do you agree with the use of "syntax" in "singular" form there?

Finally, if we all agree with the updated note text, then I'd suggested switching back to `alert.note`.

Reply:
Kathy's suggestion LGTM